src/components/App.tsx

```diff
@@ -2762,7 +2763,17 @@ class App extends React.Component<AppProps, AppState> {
         sceneY,
       );
       if (container) {
-        if (isArrowElement(container) || hasBoundTextElement(container)) {
+        if (
+          isArrowElement(container) ||
+          hasBoundTextElement(container) ||
+          !isTransparent(
+            (container as ExcalidrawTextContainer).backgroundColor,
```

Comment:
I don't like this.

Didn't know of this TS behavior. Can be fixed by changing the return type of `hasBoundTextElement` to:

```
: element is MarkNonNullable<ExcalidrawBindableElement, "boundElements">
```

Reply:
or by shuffling the checks around. But having a more precise return type in `hasBoundTextElement` doesn't hurt.

Reply:
either way, we shouldn't be asserting